models/contactModel.js

```diff
@@ -0,0 +1,37 @@
+import fs from "fs/promises";
+import path from "path";
+import { v4 as uuidv4 } from "uuid";
+import jimp from "jimp";
+import User from "#models/userModel.js";
+
+export async function updateAvatar(req, res) {
+  const userId = req.userId;
+  const { avatar } = req.files;
```

Comment:
Z tego co widze to to nie zadziała, bo brakuje sposobu dodania pliku. Dodaj go jeszcze i podeślij. :)